### DISCUSSION TITLE
Longhorn 1.7.0 install fail - which FW Rules are needed ?

### DISCUSSION THREAD

**Discussion by n00bsi** [longhorn/longhorn#9762] [General]
Hi,
need to know what Access is needed to install longhorn 1.7.0 by helm.

Opened:   

github.com/longhorn   https
charts.longhorn.io         https
objects.githubusercontent.com   https

Now I see that:   https://registry-1.docker.io/v2  is blocked...

Is there a document / list that I can send all needed access 
to the firewall team

thanks

**Comment by derekbit** [comment]
Please see https://longhorn.io/docs/1.7.2/references/networking/

> Now I see that: [registry-1.docker.io/v2](https://registry-1.docker.io/v2) is blocked...

This is public registry. You can check dockerhub's requirements.

**Comment by n00bsi** [comment]
@derekbit 

Thanks but there I did not find the urls above from my first post


>This is public registry. You can dockerhub's requirements.

You mean this?

https://auth.docker.io/

https://registry-1.docker.io/

https://index.docker.io/

https://dseasb33srnrn.cloudfront.net/

https://production.cloudflare.docker.com/

**Reply by derekbit** [reply]
yes, Longhorn images are on [registry-1.docker.io](https://registry-1.docker.io/), so you need to make sure your firewall is configured correctly to pull the images. You can check dockerhub's doc or support to see what ports are required.

**Comment by n00bsi** [comment]
thank you
